packages/mui-material/src/styles/experimental_extendTheme.js

```diff
@@ -108,30 +132,70 @@ export default function extendTheme(options = {}, ...args) {
       'Tooltip',
     ]);
     if (key === 'light') {
-      setColor(palette.Alert, 'errorColor', darken(palette.error.light, 0.6));
-      setColor(palette.Alert, 'infoColor', darken(palette.info.light, 0.6));
-      setColor(palette.Alert, 'successColor', darken(palette.success.light, 0.6));
-      setColor(palette.Alert, 'warningColor', darken(palette.warning.light, 0.6));
+      setColor(
+        palette.Alert,
+        'errorColor',
+        silent(() => darken(palette.error.light, 0.6)),
+      );
+      setColor(
+        palette.Alert,
+        'infoColor',
+        silent(() => darken(palette.info.light, 0.6)),
```

Comment:
Can we extract this in a separate function? Instead of invoking `silent(() => darken())` we could just invoke `darken()`, which will behind the scene do this. Same for the other color manipulators. We could even export these functions from the color manipulators directly by using some prefix, like `safeDarken` or something similar. My worry is that if we add new color tokens we may forget to use the `silent` function.

Reply:
Something like this?

```js
// colorManipulator.js
export const colorChannel = ...;
export const safeColorChannel = (color, warning) => {
  try {
    return colorChannel(color);
  } catch (error) {
    if (warning && process.env.NODE_ENV !== 'production') {
      console.warn(warning);
    }
    return undefined;
  }
}

// experimental_extendTheme.js
setColor(
  palette[color],
  'activeChannel',
  colorChannelSafe(colors.active, 'MUI: The value of "palette.action.active" should be one of these formats: #nnn, #nnnnnn, rgb(), rgba(), hsl(), hsla(), color().'),
);
```

Reply:
@mnajdova In case you missed the notification.

Reply:
Exactly yes, sorry for not responding sooner.